configs/yolact/README.md

```diff
@@ -0,0 +1,59 @@
+# **Y**ou **O**nly **L**ook **A**t **C**oefficien**T**s
+```
+    ██╗   ██╗ ██████╗ ██╗      █████╗  ██████╗████████╗
+    ╚██╗ ██╔╝██╔═══██╗██║     ██╔══██╗██╔════╝╚══██╔══╝
+     ╚████╔╝ ██║   ██║██║     ███████║██║        ██║
+      ╚██╔╝  ██║   ██║██║     ██╔══██║██║        ██║
+       ██║   ╚██████╔╝███████╗██║  ██║╚██████╗   ██║
+       ╚═╝    ╚═════╝ ╚══════╝╚═╝  ╚═╝ ╚═════╝   ╚═╝
+```
+
+A simple, fully convolutional model for real-time instance segmentation. This is the code for our paper:
+ - [YOLACT: Real-time Instance Segmentation](https://arxiv.org/abs/1904.02689)
+ <!-- - [YOLACT++: Better Real-time Instance Segmentation](https://arxiv.org/abs/1912.06218) -->
+
+#### For a real-time demo, check out our ICCV video:
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/0pMfmo8qfpQ/0.jpg)](https://www.youtube.com/watch?v=0pMfmo8qfpQ)
+
+# Evaluation
+Here are our YOLACT models along with their FPS on a Titan Xp and mAP on COCO's `val`:
+
+| Image Size | Backbone      | *FPS  | mAP  | Weights                                                                                                             |
+|:----------:|:-------------:|:----:|:----:|----------------------------------------------------------------------------------------------------------------------|
+| 550        | Resnet50-FPN  | 42.5 | 29.0 | [yolact_r50_epoch_55.pth](https://drive.google.com/file/d/1S30dWxxF1jmz7Tbh1cWbHZUiMpNOK1ts/view?usp=sharing) |
+| 550        | Resnet101-FPN | 33.5 | 30.4 | [yolact_r101_epoch_55.pth](https://drive.google.com/file/d/1yv752K659KbR9arOVD1f6Rst8i00OZnX/view?usp=sharing)|
```

Comment:
@chongzhou96 I have trained yolact r50 myself and the mAP is exactly the same with README.md. But single GPU training takes me around 5 days, so may I ask if you have tried SyncBN or 8-GPUs training but increasing the learning rate?
And I benchmarked the inference speed but I found it hard to get the same FPS with README.md. 
I use [`benchmark.py`](https://github.com/open-mmlab/mmdetection/blob/master/tools/benchmark.py) and it should exclude the unnecessary time of data loading and post-processing.
For `Yolact resnet 50`, on v100 I can get only 17.5 FPS. May I ask have you tried to benchmark the influence speed with this Pr?